testing/compatability/docker/README.md

```diff
@@ -0,0 +1,50 @@
+# Docker
+
+Setup docker containers with a set of Polling and Listening Deployment Targets and Workers
+
+## Pre-Req Installation
+
+Install Powershell
+
+Install Docker
+
+## Configuration
+
+Copy the file `example.config.json` to `config.json` and modify the settings
+
+- **pollingDeploymentTargets** true to set-up polling deployment targets e.g. `"pollingDeploymentTargets": true`
+- **listeningDeploymentTargets** true to set-up listening deployment targets e.g. `"listeningDeploymentTargets": true`
+- **pollingWorkers** true to set-up polling workers e.g. `"pollingWorkers": true`
+- **listeningWorkers** true to set-up listening workers e.g. `"listeningWorkers": true`
+- **tentacleVersions** comma separated list of versions to install e.g. `"tentacleVersions": "6.1.1531,6.2.277,6.3.417,6.3.605,latest"`
+
+Copy the file `example.env` to `.env` and modify the settings
+
+- **serverUrl** The url of Octopus Server to configure the Tentacles for e.g. `serverUrl=http://mymachine.local:8066`
+- **serverPollingPort** The Octopus Server polling port e.g. `serverPollingPort=10943`
+- **serverApiKey** The Octopus Server API Key e.g. `serverApiKey=API-123`
+- **environment** The environment to configure the Deployment Targets for e.g. `environment=Development`
+- **pollingRoll** The role to assign to the polling Deployment Targets e.g. `pollingRoll=pollingBreadRoll`
```

Comment:
Should `pollingRoll` and `listeningRoll` not be `pollingRole` and `listeningRole`?